Assets/02. Scripts/Customer/CustomerUI/UI_Customer.cs

```diff
@@ -0,0 +1,70 @@
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+using VInspector;
+
+public class UI_Customer : MonoBehaviour
+{
+    [Foldout("UI컴포넌트")]
+    [SerializeField]
+    private Slider _slider;
```

Comment:
인내심 수치를 보여주는 슬라이더라면, _enduranceSlider가 어떨까요
```suggestion
    [SerializeField]
    private Slider _enduranceSlider;
```